### PR TITLE
TMEDIA-906: Additional content source updates for resizer

### PR DESCRIPTION
**Pull request by nschubach**
## Description

Converting content sources to use the new resizer.

## Jira Ticket

- [TMEDIA-906](https://arcpublishing.atlassian.net/browse/TMEDIA-906)

## Acceptance Criteria

Update the following content sources to use fetch with partial caching:
- alert-bar-content-source-block
- author-content-source-block
- site-hierarchy-content-block
- tags-content-source-block

## Test Steps

1. Checkout this branch `git checkout tmedia-906`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/alert-bar-content-source-block,@wpmedia/author-content-source-block,@wpmedia/site-hierarchy-content-block,@wpmedia/tags-content-source-block`
3. http://localhost/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=alert-bar-collections. (the alert_bar collection doesn't really have any images... but this should still work)
4. http://localhost/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=author-api&config={%22slug%22:%22tariq-alshwaiki%22} (author images are not resized at this time, but should still work)
5. http://localhost/pagebuilder/tools/debugger?_website=the-gazette&type=content-debugger&source=site-service-hierarchy&config={%22sectionId%22:%22/news%22} (again, not really image dependent...)
6. http://localhost/pagebuilder/tools/debugger?_website=daily-intelligencer&type=content-debugger&source=tags-api&config={%22slug%22:%22test-ta%22} (no images here, but should work with published story tags)

## Author Checklist
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/tags-content-source-block/sources/tags-api.js [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
+			const error = new Error("Not found");
+			error.statusCode = 404;
+			return Promise.reject(error);
+		});
+};
+
+export default {
+	fetch,
+	params,
+	schemaName: "ans-feed",
```
Comment (matthewroach) [inline blocks/tags-content-source-block/sources/tags-api.js]: Tags API is does not return ANS items. Setting this `schemaName` here will enable this content source to be available within some of our blocks and would not work for the user. We can remove this `schemaName` value and leave off as per previous implementation

**Review by matthewroach — CHANGES_REQUESTED** [review]
One minor change @nschubach, otherwise all look good

File: blocks/tags-content-source-block/sources/tags-api.js (hunk 1 above)
Comment (nschubach) [reply blocks/tags-content-source-block/sources/tags-api.js]: @matthewroach : given that this is not an ANS item, do we really need to check for images and add resizer auth?  (that goes for some of the others as well)

Comment (nschubach) [inline blocks/tags-content-source-block/sources/tags-api.js]:
```suggestion
```

Comment (matthewroach) [reply blocks/tags-content-source-block/sources/tags-api.js]: Probably not no - we should only need to update content sources that are doing the resizer logic today

**Review by matthewroach — APPROVED** [review]

**Pull request merged**